src/backend/EdFi.DataManagementService.Backend.Postgresql.Test.Integration/DatabaseTestWithRollback.cs

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using Npgsql;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using Respawn;
+
+namespace EdFi.DataManagementService.Backend.Postgresql.Test.Integration;
+
+/// <summary>
+/// Implements a database cleanup after each test
+/// </summary>
+[AttributeUsage(AttributeTargets.All)]
+public class DatabaseTestWithRollback : Attribute, ITestAction
+{
+    private static readonly string _connectionString =
+        "host=localhost;port=5432;username=postgres;password=abcdefgh1!;database=edfi_datamanagementservice_test_integration;Application Name=EdFi.DataManagementService.Backend.Postgresql.Test.Integration";
```

Comment:
Right now this is specific to my dev env. Tests are marked ignored for now. Will be enabled by DMS-188